src/render.js

```diff
@@ -0,0 +1,73 @@
+import {
+  city, cord, temp, hum, status,
+} from './extractInfo';
+
+function clear(container) {
+  while (container.firstChild) {
+    container.removeChild(container.firstChild);
+  }
+}
```

Comment:
- [ ] Please change this also to es6 arrow function syntax, in keeping up with important trends.